torchx/schedulers/ray/ray_driver.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import json
+import dataclasses
+import ray
+from typing import Any, Dict, Iterable, List, Optional, Set, Type
+import subprocess
+from .ray_common import RayActor
+
+@ray.remote
+class CommandActor:
+    def __init__(self, train_script):
+        self.train_script = train_script
+
+    def run_command(self):
+        return exec(open(self.train_script))
```

Comment:
@cbalioglu `exec(open(self.train_script))` should work here since will `open` will read the file and convert it to string form that `exec` can now execute.

The benefit of this vs. `os.execve` or `subprocess` is that there's no extra logic needed to handle exceptions or stdout/stderr. Any exception that's raised by the user script will be treated as an exception occurring in the actor itself, since the user script is being run in the same process as the actor. This also means we don't need to worry about exit codes.